slices/ca-certificates.yaml

```diff
@@ -43,15 +25,33 @@ slices:
       ]
       content.write("/etc/ssl/certs/ca-certificates.crt", "".join(certs))
 
+  # This slice offers a standard chunk of the ca-certificates package.
+  # It contains the default certificates at /usr/share/ca-certificates/mozilla,
+  # a certificates bundle at /etc/ssl/certs/ca-certificates.crt and a config
+  # file at /etc/ca-certificates.conf.
+  data-with-certs:
+    essential:
+      - ca-certificates_data
+    contents:
+      # This config file is generated by the deb's maintainer scripts.
+      /etc/ca-certificates.conf: {text: FIXME, mutable: true}
+      /usr/share/ca-certificates/mozilla/**:
+    mutate: |
+      certs_dir = "/usr/share/ca-certificates/mozilla/"
+      names = [
+        "mozilla/{}".format(name) for name in content.list(certs_dir)
+      ]
+      content.write("/etc/ca-certificates.conf", "\n".join(names))
+
   bins:
     essential:
       - openssl_bins
       # The `/usr/sbin/update-ca-certificates` script requires sed to run.
       - sed_bins
     contents:
       # To run update-ca-certificates without breaking the default certificates
-      # at /etc/ssl/certs/ca-certificates.crt, the `_data` slice should also be
-      # included. For details, see the `update-ca-certificates` script.
+      # at /etc/ssl/certs/ca-certificates.crt, the `_data-with-certs` slice should
```

Comment:
Let's address #285 here.

Reply:
Note the slice name is changed. The `_data-with-certs` is therefore not a bad slice reference anymore.

Reply:
Ah okay. Feel free to close the issue once this PR gets merged then.